src/lib/snack-bar/snack-bar-container.ts

```diff
@@ -110,7 +111,8 @@ export class MdSnackBarContainer extends BasePortalHost implements OnDestroy {
   }
 
   /** Attach a template portal as content to this snack bar container. */
-  attachTemplatePortal(): Map<string, any> {
+  /* tslint:disable-next-line */
```

Comment:
Yes, we can do that but this will not be inline with other `attachTemplatePortal` calls that does accept a type parameter.

This is not the only reason, main reason is that right now we need to add the `tslint disable` but once we move to TypeScript 2.4 it will be easy to migrate it to:

```ts
attachTemplatePortal<C = any>(): EmbeddedViewRef<C> {
      throw Error('Not yet implemented');
}
```

This way we allow setting the type parameter for those who want.
Later we will support those who "doesnt" want as well :)